File: Modul-6/Soal-1/PRAK601-2410817320001-NazlaSalsabila.py
def print_matrix(baris, kolom, data):
    index = 0
    for i in range(baris):
        for j in range(kolom):
            if j != kolom - 1:
                print(data[index], end=" ")
            else:
                print(data[index])
            index += 1

print("Masukkan jumlah baris dan kolom:", end=" ")
baris, kolom = map(int, input().split())
print("Masukkan angka-angka dalam matriks:", end=" ")
data = list(map(int, input().split()))
print_matrix(baris, kolom, data)

File: Modul-6/Soal-1/PRAK601-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int baris, kolom;
    printf("Masukkan jumlah baris dan kolom: ");
    scanf("%d %d", &baris, &kolom);
    
    int data[baris * kolom];
    printf("Masukkan angka-angka dalam matriks: ");
    for (int i = 0; i < baris * kolom; i++) {
        scanf("%d", &data[i]);
    }
    
    int index = 0;
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d", data[index++]);
            if (j < kolom - 1) printf(" ");
        }
        printf("\n");
    }
    
    return 0;
}